src/btstream.cpp

```diff
@@ -25,6 +25,8 @@ namespace oxen::quic
             ep = get_location(data, btlc.consume_string_view());
 
         req_body = get_location(data, btlc.consume_string_view());
+
+        btlc.finish();
```

Comment:
I dunno about this change i really need to think about it.

@tewinget thoughts?